includes/core/server/request/trait-level3.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Trait file for WCPay\Core\Server\Request\Intention.
```

Comment:
```suggestion
 * Trait file for WCPay\Core\Server\Request\Level3.
```

Reply:
Done. Addressed in https://github.com/Automattic/woocommerce-payments/pull/5136/commits/8f1aa8fd35a4f80de54b317d8892d0aae95753cd